src/LibVLCSharp.Forms/Shared/Themes/Generic.xaml

```diff
@@ -239,71 +206,151 @@
-                                    <Label HorizontalOptions="CenterAndExpand" x:Name="TrackBarLabel" TextColor="White" VerticalOptions="CenterAndExpand" />
-                                </Frame>
-                            </local:SwipeToUnLockView.TrackBar>
+                                <local:SwipeToUnLockView.TrackBar>
+                                    <Frame Padding="0" BackgroundColor="#1C1C1C" CornerRadius="30" HasShadow="false">
+                                        <Label HorizontalOptions="CenterAndExpand" x:Name="TrackBarLabel" TextColor="White" VerticalOptions="CenterAndExpand" />
+                                    </Frame>
+                                </local:SwipeToUnLockView.TrackBar>
 
-                            <local:SwipeToUnLockView.FillBar>
-                                <Frame Padding="0" BackgroundColor="#1C1C1C" CornerRadius="20" HasShadow="false" />
-                            </local:SwipeToUnLockView.FillBar>
-                        </local:SwipeToUnLockView>
+                                <local:SwipeToUnLockView.FillBar>
+                                    <Frame Padding="0" BackgroundColor="#1C1C1C" CornerRadius="20" HasShadow="false" />
+                                </local:SwipeToUnLockView.FillBar>
+                            </local:SwipeToUnLockView>
+                        </StackLayout>
                     </StackLayout>
-                    
-                </StackLayout>
+
+                    <!-- Overlay -->
+                    <ContentView HeightRequest="300" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" x:Name="TracksOverlayView" AbsoluteLayout.LayoutBounds="0, 0, 1, 1"  AbsoluteLayout.LayoutFlags="All" IsVisible="False" BackgroundColor="#7F000000">
+                        <ScrollView VerticalScrollBarVisibility="Never" HorizontalOptions="FillAndExpand" VerticalOptions="Fill" Orientation="Vertical">
+                            <Grid>
+                                <StackLayout Grid.Row="0" HorizontalOptions="FillAndExpand">
+                                    <Label Style="{StaticResource TrackTypeLabelStyle}" x:Name="AudioTracksLabel" />
+                                    <ListView x:Name="AudioTracksListView" ItemTemplate="{StaticResource TrackViewCellDataTemplate}" Style="{StaticResource TracksListViewStyle}" />
+                                </StackLayout>
+                                <StackLayout Grid.Row="1" HorizontalOptions="FillAndExpand">
+                                    <Label Style="{StaticResource TrackTypeLabelStyle}" x:Name="VideoTracksLabel" />
+                                    <ListView x:Name="VideoTracksListView" ItemTemplate="{StaticResource TrackViewCellDataTemplate}" Style="{StaticResource TracksListViewStyle}" />
+                                </StackLayout>
+                                <StackLayout Grid.Row="2" HorizontalOptions="FillAndExpand">
```

Comment:
Wondering if we shouldn't hide these if there are 0 subtitle tracks (or 0 audio/video tracks, unlikely for video at least).
But then one is dynamically added, it should appear again. What do you think?

The benefit might not be worth the added complexity though.

Reply:
> if one is dynamically added, it should appear again
Doesn't seem complex if it's a data binding with a Converter

Can tracks be added dynamically ? If so, can they be removed ? If it can be removed, what's the stability of an index-based track identification?

Reply:
> Can tracks be added dynamically ?  If so, can they be removed ? 

The user could add an external .srt file for example. I think some streams can dynamically add/remove ES too...

>  If it can be removed, what's the stability of an index-based track identification?

That's fixed in v4 (one of the reasons this PR won't apply nicely to master)
https://github.com/videolan/libvlcsharp/blob/a6816208f29792685373eb556034721ac0684681/src/LibVLCSharp/Structures/MediaTrack.cs#L23

Reply:
> Wondering if we shouldn't hide these if there are 0 subtitle tracks (or 0 audio/video tracks, unlikely for video at least).
> But then, if one is dynamically added, it should appear again. What do you think?

Good idea ! I am currently updating the PR :-)